mayastor/src/rebuild.rs

```diff
@@ -60,7 +62,24 @@ impl fmt::Display for RebuildState {
     }
 }
 
-#[derive(Debug)]
+struct TaskResult {
+    blk: u64,
+    id: u64,
+    error: Option<RebuildError>,
+}
+
+const SEGMENT_TASKS: u64 = 4;
+const SEGMENT_SIZE: u64 = 10 * 1024; // 10KiB
+
+struct RebuildTasks {
+    buffers: Vec<DmaBuf>,
```

Comment:
Likewise, please try to comment all structs and members, and functions.. it really pays of to do this consistently. you can in fact even tell the compiler to help you by setting that you do not allow missing docs. 

iirc its `#[deny(missing_docs)]`

Reply:
yep I should have and will do indeed. I'll add that attribute, that'll be quite handy :+1: